packages/@react-aria/overlays/test/calculatePosition.test.js

```diff
@@ -37,6 +37,9 @@ const containerDimensions = {
   left: 0
 };
 
+// Reset JSDom's default margin on the body
+document.body.style.margin = 0;
```

Comment:
would this be useful to do globally during test setup instead?

Reply:
idk how to do that